src/main/java/com/aws/greengrass/clientdevices/auth/CertificateManager.java

```diff
@@ -126,18 +127,24 @@ public void stopMonitors() {
     }
 
     /**
-     * Return a list of CA certificates used to issue client certs.
+     * Returns a singleton list with the PEM encoded CA at position 0 of the CA chain.
      *
-     * @return a list of CA certificates for issuing client certs
      * @throws KeyStoreException            if unable to retrieve the certificate
      * @throws IOException                  if unable to write certificate
      * @throws CertificateEncodingException if unable to get certificate encoding
+     *
+     * @deprecated use getX509CACertificates
      */
     public List<String> getCACertificates() throws KeyStoreException, IOException, CertificateEncodingException {
         return Collections.singletonList(CertificateHelper.toPem(certificateStore.getCACertificate()));
     }
 
-    private X509Certificate[] getX509CACertificates() throws KeyStoreException {
+    /**
+     * Returns the full CA chain.
+     *
+     * @throws KeyStoreException when failing to read certificates from key store.
+     */
+    public X509Certificate[] getX509CACertificates() throws KeyStoreException {
```

Comment:
Why make changes to CertManager at this point? Can we not just directly call the CertStore?

Reply:
I would have to change the useCase and add an additional dependency which means more changes. I was just focusing on the functionality. I can do it though